test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestSettingsContext.cs

```diff
@@ -191,5 +192,20 @@ public void AddNetStandardFeeds()
 
             Save();
         }
+
+        // Simply add any text as section into nuget.config file, adding large child node into nuget.config via api is tedious.
+        public static void AddSectionIntoNuGetConfig(string path, string content, string parentNode)
```

Comment:
Creating a utility method to simplify/deduplicate code is great.  Trying to create a general-purpose utility method with only 1 scenario in use is risky.

Should this method guard against adding a section when a like-named section already exists?

Reply:
Thank you for your feedback. 
I added guard against adding again like-named section already exists, please check now. But I didn't add unit test for asserting it, since we don't have specific file for adding test for `test utility` itself. Alternatively, I could add one DotnetVerifyTests.cs, but not sure if it's correct thing to do.